drivers/storage/portworx/component/pvccontroller.go

```diff
@@ -304,10 +304,13 @@ func (c *pvcController) createDeployment(
 		"--controllers=persistentvolume-binder,persistentvolume-expander",
 		"--use-service-account-credentials=true",
 	}
-	if pxutil.IsOpenshift(cluster) {
-		command = append(command, "--leader-elect-resource-lock=endpoints")
-	} else {
-		command = append(command, "--leader-elect-resource-lock=configmaps")
+
+	if c.k8sVersion.LessThan(k8sutil.K8sVer1_24) {
```

Comment:
what about k8s 1.24? this arg is no longer needed?

Reply:
I think there is a default for the leader election lock type in k8s 1.24. It defaults to `leases`.

Reply:
Yes, it has default value in 1.24. not sure if it has a default value before though as i could not find command help on specific version